.github/workflows/docker.yml

```
@@ -2,13 +2,17 @@ name: Docker Build and Push
 # Build & Push builds the simapp docker image on every push to master
 # and pushes the image to https://hub.docker.com/u/provenanceio
 on:
+  # Uncomment this trigger if you'd like to test this workflow during active development.
+  # This trigger is configured to not push any images to Docker Hub.
+  pull_request:
```

Rather than comment/uncomment, for this, let's include the paths of the files involved.

E.g. 
```suggestion
  pull_request:
    paths:
      - "docker/blockchain/Dockerfile"
      - ".github/workflows/release.yml"
```

If there's another file in our repo used by the heighliner stuff, include that in the `paths` up there too.

That'll make this only run for PRs with something involved in these processes are being updated. It'll still run when any PR is merged to main, or when a version tag is created.